src/api/api.tsx

```diff
@@ -35,10 +46,31 @@ export default class API {
     });
   }
 
-  addPdfs(pdfs: FormData) {
+  addPdfs(pdf: File, vacancy: string) {
+    const pdfs = new FormData();
+    pdfs.append('cv', pdf);
+    pdfs.append('vacancy', vacancy);
+  
     return this.fetchAdvanced(this.addPdfsURL(), {
       method: 'POST',
       body: pdfs,
     });
   }
+
+  // Fetch vacancies
+  fetchVacancies() {
+    return this.fetchAdvanced(this.vacanciesURL(), {});  
+  }
+
+  // Get vacancy by ID
+  getVacancy(vacancyId: string) {
+    return this.fetchAdvanced(this.getVacancyURL(vacancyId), {});  
+  }
+
+  // Fetch applicants by vacancy ID
+  fetchApplicants(vacancyId?: string) {
```

Comment:
```suggestion
  fetchApplicants(vacancyId: string) {
```
Remove '?' because it makes the vacancyId parameter optional, which it should not be in this case.